topics/earth-system/tutorials/ocean-data-view/tutorial.md

```diff
@@ -0,0 +1,250 @@
+> {: .tip}
+>
+{: .hands_on}
+
+> <details-title> Some complementary information about your data </details-title>
+> Argo is an international program to observe the interior of the ocean with a fleet of profiling floats drifting in the deep ocean currents (check it [here](https://argo.ucsd.edu)). Argo GDAC is a dataset of 5 billion in situ ocean observations from 18.000 profiling floats (4.000 active) which started 20 years ago. Argo GDAC dataset is a collection of 18.000 NetCDF files. It is a major asset for ocean and climate science, a contributor to IOCCP reports. For more informations click [here]( https://doi.org/10.17882/42182)
+>
+> These data are available on AWS registry of Open Data. This registry exists to help people discover and share datasets that are available via AWS resources. [See recent additions and learn more about sharing data on AWS](https://registry.opendata.aws/).
+{: .details}
+
+# Ocean Data View
+## Map your data
+
+> <tip-title>Copy pasting between computer and ODV</tip-title>
+> You can expand the ODV left panel (where there are 3 dots, vertically) to access the "clipboard" menu, and paste the content you want to paste on a ODV form. From there you can copy paste everything from one side to the other. Then, click outside of this panel to collapse it.
+> 
+> ![Image showing in transparent on the left of the ODV interface the clipboard](../../images/coastal_water_dyn/clipboard.png)
+{: .tip}
+
+> <tip-title>ODV - Disconnected</tip-title>
+> If at one point your ODV interface becomes grey with a red pannel "X ODV - Disconnected", do NOT panik ;) you just need to reload your tab (circular arrow top left)
+{: .tip}
+
+> <hands-on-title>Loading Argo data</hands-on-title>
+>
+> 1. Go to the top left of ODV and click on "Open..."
+> 2. Click on "1900392_prof.nc" of the pop-up screen and on  **Open**
+> 3. Press **Next >** for "Select Dimensions (Step 1 of 4)"
+> 4. Press **Next >** for "Associate Meta Variables (Step 2 of 4)"
+> 5. Press **Next >** for "Select Primary Variable (Step 3 of 4)"
+> 6. Press **Finish** for "Subset Dimensions (Step 4 of 4)"
+> 7. "Enter netCDF Setup Name:" write "argo_prof_data", then, press **OK**
+{: .hands_on}
+![Map of all the stations of the Argo glider](../../images/coastal_water_dyn/argo_stations.png)
+
+> <question-title></question-title>
+>
+> 1. What are the longitude and latitude of the red dot ?
+>
+> > <solution-title></solution-title>
+> >
+> > 1. On the to right window you can read Longitude 7°E and Latitude 21°S.
+> >
+> {: .solution}
+>
+{: .question}
+
+## Create profiles curves
+
+> <hands-on-title>Create a multiple windows view</hands-on-title>
+> 1. On the top left press **+**
+> 2. Click on **2 Station Windows**
+> ![Image showing where to go to have the 2 STATION Windows](../../images/coastal_water_dyn/2_station.png)
```

Comment:
```suggestion
>    ![Image showing where to go to have the 2 STATION Windows](../../images/coastal_water_dyn/2_station.png)
```

you should indent images to match the start of the text, otherwise it will interrupt your numbered list, and you'll see `1. ... 2.... [image] 1...` which probably isn't what you wanted